client/landing/stepper/declarative-flow/internals/steps-repository/processing-step/style.scss

```diff
@@ -99,14 +99,3 @@
 	}
 }
 
-
-.is-section-stepper .newsletter.processing {
-	.processing-step {
-		.loading-bar {
-			background-color: var(--studio-blue-10);
-			&::before {
-				background: var(--studio-blue-50);
-			}
-		}
-	}
-}
```

Comment:
Moved to `client/landing/stepper/declarative-flow/internals/steps-repository/processing-step/tailored-flow-precheckout-screen/style.scss` instead where other modifications to this step are as well.